### ISSUE TITLE
`https://tarballs.nixos.org/imlib-1.9.15.tar.gz` is 404

### ISSUE THREAD

**Issue by kvtb** [NixOS/nixpkgs#138213]
https://github.com/NixOS/nixpkgs/blob/586a9e6bffb1b4e0e444bf4013169d8f415f2987/pkgs/development/libraries/imlib/default.nix#L6-L11

Steps to reproduce

```
wget https://tarballs.nixos.org/imlib-1.9.15.tar.gz
```

**Comment by marsam** [comment]
I think we can replace it with https://download.gnome.org/sources/imlib/1.9/imlib-1.9.15.tar.gz. Would you mind creating a PR with this change?

**Comment by kvtb** [comment]
The problem is disappearance from `nixos.org`

**Comment by marsam** [comment]
True, but it probably should be reported on https://github.com/NixOS/nixos-org-configurations
@zimbatm maybe it's something related to https://github.com/NixOS/nixos-org-configurations/issues/161?

Nonetheless, I still think we should use the Gnome source, because it's also used by other repositories

**Comment by zimbatm** [comment]
It's weird, I don't see this object in the S3 bucket. It seems like this issue is affecting other packages like pdf2xml, mk, keen4, azureus, aefs, xpf.
openjdk which is loading the object from a sub-folder is fine, it's only the top-level ones that are missing.
